modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavascriptApolloClientCodegen.java

```diff
@@ -104,6 +104,10 @@ public JavascriptApolloClientCodegen() {
 
         modifyFeatureSet(features -> features.includeDocumentationFeatures(DocumentationFeature.Readme));
 
+        generatorMetadata = GeneratorMetadata.newBuilder(generatorMetadata)
+                .stability(Stability.DEPRECATED)
+                .build();
+
```

Comment:
In this file, can you please also rename `javascript-apollo` to `javascript-apollo-deprecated`?